features/step_definitions/checks_for_rails_security_issues_steps.rb

```diff
@@ -19,5 +19,6 @@ def rails_security_output
 end
 
 Given(/^I am on project directory$/) do
-  @dirs = ["tmp/aruba/#{@projectname}"]
+  dir = Aruba::ArubaPath.new("tmp/aruba/#{@projectname}")
+  aruba.instance_variable_set('@current_directory', dir)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.